samples/interactive-tutorials/prediction/symple_prediction_py.md

```diff
@@ -0,0 +1,109 @@
+<walkthrough-metadata>
+  <meta name="title" content="Getting The Recommendations. Prediction Service" />
+  <meta name="description" content="Prediction service features" />
+  <meta name="component_id" content="593554" />
+  <meta name="short_id" content="true" />
+</walkthrough-metadata>
+
+# Getting The Recommendations. Prediction service features
+
+## Introduction
+
+This is the third part of the complex tutorial "Getting The Recommendations".
+
+Before you can request predictions from Recommendations AI, you need a trained and tuned recommendation model and one or more active serving configurations.
+
+//TODO - add the linc to the tutorial
+To learn how to prepare the data in a Retail catalog and to ingest the historical user events, go through the tutorial <...>
+
+To create and train a recommendation model, proceed with the video tutorial: <...>
+
+The detailed information about the recommendation service could be found in the [Retail API documentation](https://cloud.google.com/retail/docs/predict#recommendations-predict-java).
+
+<walkthrough-tutorial-duration duration="10"></walkthrough-tutorial-duration>
+
+<<_shared/_set_up_env_python.md>>
+
+## Prediction service. Simple request
+
+1. The `PredictRequest` request has the following fields:
+    - `placement`—the ID of the Recommendations AI placement. Before you can request predictions from your model, you must create at least one placement for it.
+    - `user_event`—context about the user, what they are looking at and what action they took to trigger the predict request.
+    - `page_size`—maximum number of results to return per page. If zero, the service will choose a reasonable default. The maximum allowed value is 100.
+    - `page_token`—the previous PredictResponse.next_page_token.
+    - `filter`—filter for restricting prediction results with a length limit of 5,000 characters. The filter can be applied to the field `tags` or can be used the `filterOutOfStockItems` flag.
+    - `params`—additional domain specific parameters for the predictions.
+    - `labels`—the labels applied to a resource. More about the labels could be found in the [documentation](https://cloud.google.com/resource-manager/docs/creating-managing-labels#requirements).
+   
+//TODO check the file path
+1. Open <walkthrough-editor-select-regex filePath="cloudshell_open/python-retail/samples/interactive-tutorials/prediction/get_prediction.py" regex="# get prodciction request">prediction/get_prediction.py</walkthrough-editor-select-regex> file and check the `PredictRequest` request.
```

Comment:
`1. Open the <walkthrough-editor-select-regex filePath="cloudshell_open/python-retail/samples/interactive-tutorials/prediction/get_prediction.py" regex="# get prodciction request">prediction/get_prediction.py</walkthrough-editor-select-regex> file and check the `PredictRequest` request.`